content/blog/2022-01-28-volcengine-violates-aplv2/index.md

```diff
@@ -45,4 +45,40 @@ Also, we can't find anything on their website to declare they are distributing S
 
 **All above have proved they are violating the Apache 2.0 License, and don't respect Apache Software Foundation and Apache SkyWalking's IP and Branding**.
 
-We have contacted their legal team, and wait for their official response.
\ No newline at end of file
+We have contacted their legal team, and wait for their official response.
+
+## Resolution
+
+On Jan. 29th night, UTC+8, 2022. We received a response from Volcengine's APMPlus team. They admitted their violation behaviors, and made the following changes.
```

Comment:
```suggestion
On Jan. 30th night, UTC+8, 2022. We received a response from Volcengine's APMPlus team. They admitted their violation behaviors, and made the following changes.
```

@apmplus I think this feedback is officially working on 30th, including the private mail list response.